src/Data.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Title from "./components/Title";
```

Comment:
too many components for the little amount of information your using, but i like all of the work, just leave some comments please and thank you